projects/1.20-fabric/assets/great-big-world/great_big_world/lang/zh_cn.json

```
@@ -0,0 +1,292 @@
+    "biome.great_big_world.island_sparse_jungle": "岛屿稀疏丛林",
+    "biome.great_big_world.volcanic_beach": "火山沙滩",
+    "biome.great_big_world.volcanic_slopes": "火山山坡",
+    "biome.great_big_world.volcanic_crater": "火山口",
+  
+    "enchantment.great_big_world.diluting": "稀释",
+    "enchantment.great_big_world.poison_gaze": "淬毒",
+    "enchantment.great_big_world.illuminating": "照明",
+  
+    "death.attack.moose": "%1$s被驼鹿杀死了",
+  
+    "great_big_world.subtitles.entity.moose.hurt": "驼鹿：受伤",
+    "great_big_world.subtitles.entity.moose.death": "驼鹿：死亡",
+    "great_big_world.subtitles.entity.moose.warning": "驼鹿：咆哮",
+  
+    "advancements.adventure.on_a_tropical_island.title": "热带岛屿",
+    "advancements.adventure.on_a_tropical_island.description": "登陆一个热带岛屿",
+    "advancements.adventure.oxidizing_nautilus.title": "鹦鹉螺氧化",
+    "advancements.adventure.oxidizing_nautilus.description": "用鹦鹉螺来氧化铜块",
+    "advancements.adventure.a_mask_of_wood.title": "木制的面具",
+    "advancements.adventure.a_mask_of_wood.description": "获得一个木制面具",
+    "advancements.adventure.find_lighthouse.title": "那不是信标",
+    "advancements.adventure.find_lighthouse.description": "找到一个灯塔",
+    "advancements.adventure.find_swamp_pyramid.title": "上古遗迹",
+    "advancements.adventure.find_swamp_pyramid.description": "找到一个沼泽金字塔",
+  
+    "text.great_big_world.config.title": "Great Big World配置",
+    "text.great_big_world.config.general": "常规设置",
+    "text.great_big_world.config.option.bambooTorchesOnLeaves": "竹火把可放置在树叶上",
+    "text.great_big_world.config.option.bambooTorchesOnLeaves.@Tooltip": "是否允许竹火把放置在树叶上。",
+    "text.great_big_world.config.option.masksAngerEndermen": "木制面具激怒末影人",
+    "text.great_big_world.config.option.masksAngerEndermen.@Tooltip": "装备木制面具时是否能激怒末影人。",
+    "text.great_big_world.config.option.pillagersAttackMoose": "掠夺者对驼鹿的攻击性",
+    "text.great_big_world.config.option.pillagersAttackMoose.@Tooltip": "掠夺者是否应该主动攻击驼鹿。",
+    "text.great_big_world.config.option.wolvesAttackMoose": "狼对驼鹿的攻击性",
+    "text.great_big_world.config.option.wolvesAttackMoose.@Tooltip": "狼是否应该主动攻击驼鹿。",
+    "text.great_big_world.config.option.maxDistanceForRootConversion": "火山砂转化范围",
+    "text.great_big_world.config.option.maxDistanceForRootConversion.@Tooltip": "火山砂将泥土转化为缠根泥土的最大范围。",
+    "text.great_big_world.config.option.shedAntlerBaseRegrowTime": "鹿角再生速度",
+    "text.great_big_world.config.option.shedAntlerBaseRegrowTime.@Tooltip": "驼鹿鹿角脱落和再生的所需时间。",
+    "text.great_big_world.config.option.nautilusOxidizeChance": "鹦鹉螺氧化几率",
+    "text.great_big_world.config.option.nautilusOxidizeChance.@Tooltip": "鹦鹉螺氧化铜块和海晶石的几率。",
+    "text.great_big_world.config.option.dilutingModifier": "稀释效果配置",
+    "text.great_big_world.config.option.dilutingModifier.@Tooltip": "稀释效果减少其他状态效果持续时间的百分比。",
+    "text.great_big_world.config.option.auraEffectTransferModifier": "光环型药水配置",
+    "text.great_big_world.config.option.auraEffectTransferModifier.@Tooltip": "光环型药水效果在转移后持续时间的百分比。",

```

```suggestion
    "text.great_big_world.config.option.auraEffectTransferModifier.@Tooltip": "酿造为光环型药水效果后持续时间的百分比。",
```
整个和Minecraftwiki上喷溅型一样的动词（）